lib/os/Kconfig

```diff
@@ -38,6 +43,14 @@ config MPSC_PBUF
 	  storing variable length packets in a circular way and operate directly
 	  on the buffer memory.
 
+config ONOFF
+	bool "On-Off Manager"
+	select NOTIFY
```

Comment:
in general, `depends on` should be preferred as per https://docs.zephyrproject.org/latest/build/kconfig/tips.html#select-recommendations.